stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../assets/scss/mixins';
+@import '../../assets/scss/breakpoints';
+@import '../../assets/scss/variables';
+
+#toggleSearch {
```

Comment:
@Ashish-srij Please don't use the camelcasing in class and id names

Reply:
Done